styles/style.css

```diff
@@ -0,0 +1,66 @@
+/* style.css */
+
+.first-container {
+    max-height: 700px;
+    
+    border: lightblue 2px solid;
+    display: grid;
+    grid-template: 100% / 50%, 50%;
+}
+
+.first-container ul {
+    grid-column-start: 2;
+    line-height: 2;   
+}
+
+.first-container p {
+    max-height: 500px;
+    font-size: xx-large;
+    font-style: italic;
+    color: blue;
+    line-height: 3;
+    
+}
```

Comment:
If we want text with a large font size, then let's use a heading, like `h2` or `h3` instead of a `p` tag